demos/twa-location-delegation/README.md

```diff
@@ -0,0 +1,7 @@
+# Trusted Web Activity / Location Delegation Demo
+
+This demo application shows how to make the TWA app uses Android location permission instead of
```

Comment:
nits:
uses -> use
Android ->Android's
browser -> the browser's

Reply:
Done